src/coreclr/inc/cfi.h

```diff
@@ -9,7 +9,9 @@ enum CFI_OPCODE
 {
    CFI_ADJUST_CFA_OFFSET,    // Offset is adjusted relative to the current one.
    CFI_DEF_CFA_REGISTER,     // New register is used to compute CFA
-   CFI_REL_OFFSET            // Register is saved at offset from the current CFA
+   CFI_REL_OFFSET,           // Register is saved at offset from the current CFA
+   CFI_DEF_CFA,              // Used by ILC, not emitted by JIT
```

Comment:
```suggestion
   CFI_DEF_CFA,              // Take address from register and add offset to it.
```
All CFI opcodes are only used by AOT compilation